<launch>
  <node pkg="final_challenge_2022" name="drive" type="drive.py" output="screen">
  	<param name="drive_topic" value="/drive" />
  </node>

  <node pkg="final_challenge_2022" name="line_detector" type="line_detector.py" output="screen">
  </node>

  <node pkg="final_challenge_2022" name="homography_transformer" type="homography_transformer.py" output="screen">
  </node>
</launch>
